eagle-core/eagle-data-process/eagle-stream-process-api/src/test/java/org/apache/eagle/datastream/TestExecutionEnvironmentJava.java

```diff
@@ -0,0 +1,25 @@
+package org.apache.eagle.datastream;
```

Comment:
Thanks for the review and I will fix them.

I have integrated: http://creadur.apache.org/rat/apache-rat-plugin/
We should make sure passing RAT checking in the CI environment:
    
    mvn apache-rat:check

before merging any pull request.